stormpath/src/main/java/com/stormpath/sdk/StormpathConfiguration.java

```diff
@@ -0,0 +1,175 @@
+package com.stormpath.sdk;
+
+public class StormpathConfiguration {
+
+    private String baseUrl;
+
+    private String oauthPath;
+
+    private String registerPath;
+
+    private String passwordResetPath;
+
+    private String verifyEmailPath;
+
+    private String logoutPath;
+
+    private String userProfilePath;
+
+    StormpathConfiguration(Builder builder) {
+        baseUrl = normalizeUrl(builder.baseUrl);
+        oauthPath = normalizePath(builder.oauthPath);
+        registerPath = normalizePath(builder.registerPath);
+        passwordResetPath = normalizePath(builder.passwordResetPath);
+        verifyEmailPath = normalizePath(builder.verifyEmailPath);
+        logoutPath = normalizePath(builder.logoutPath);
+        userProfilePath = normalizePath(builder.userProfilePath);
+    }
+
+    String baseUrl() {
+        return baseUrl;
+    }
+
+    String oauthPath() {
+        return oauthPath;
+    }
+
+    String registerPath() {
+        return registerPath;
+    }
+
+    String passwordResetPath() {
+        return passwordResetPath;
+    }
+
+    String verifyEmailPath() {
+        return verifyEmailPath;
+    }
+
+    String logoutPath() {
+        return logoutPath;
+    }
+
+    String userProfilePath() {
+        return userProfilePath;
+    }
+
+    String oauthUrl() {
+        return baseUrl + oauthPath;
+    }
+
+    String registerUrl() {
+        return baseUrl + registerPath;
+    }
+
+    String passwordResetUrl() {
+        return baseUrl + passwordResetPath;
+    }
+
+    String verifyEmailUrl() {
+        return baseUrl + verifyEmailPath;
+    }
+
+    String logoutUrl() {
+        return baseUrl + logoutPath;
+    }
+
+    String userProfileUrl() {
+        return baseUrl + userProfilePath;
+    }
+
+
+    private static String normalizePath(String path) {
+        if (!path.startsWith("/")) {
+            path = "/" + path;
+        }
+        return path;
+    }
+
+    private static String normalizeUrl(String url) {
+        if (url.endsWith("/")) {
+            url = url.substring(0, url.length() - 1);
+        }
+        return url;
+    }
+
+    public static class Builder {
+
+        String baseUrl;
+
+        String oauthPath = "/auth/token";
+
+        String registerPath = "/register";
+
+        String passwordResetPath = "/forgot";
+
+        String verifyEmailPath = "/verify";
+
+        String logoutPath = "/logout";
+
+        String userProfilePath = "/me";
+
+        public Builder baseUrl(String baseUrl) {
```

Comment:
The Android SDK is currently only known to work with express-stormpath, I'm not sure if we can set this as default. @edjiang @omgitstom what do you guys think?